pro-planner-client/src/components/AddOptionForm.jsx

```diff
@@ -0,0 +1,59 @@
+import React, {useState} from 'react';
+import {Modal, Button, Form} from "react-bootstrap";
+import {useDispatch} from 'react-redux';
+import {addOption} from "../redux/pollSlice";
+import {v4 as uuidv4} from 'uuid';
+
+function AddOptionForm({poll, showModal, setShowModal}) {
+    const [newOption, setNewOption] = useState("")
+    const dispatch = useDispatch();
+
+    let formResult = {
+        pollId: poll.pollId,
+        optionId: uuidv4(),
```

Comment:
Not entirely sure the necessity of IDs for different options/polls. @vincewu124 if you have some insights on how this might work with the backend/db, that'd be great.